task bynary search/script.js

```diff
@@ -0,0 +1,59 @@
+function binarySearch(array, element, first, last) {
```

Comment:
do you really need `first` and `last` function arguments in non-recursive implementation?